app/src/main/java/com/github/polypoly/app/network/FakeRemoteStorage.kt

```diff
@@ -2,18 +2,65 @@ package com.github.polypoly.app.network
 
 import android.os.Build
 import androidx.annotation.RequiresApi
-import com.github.polypoly.app.game.Skin
-import com.github.polypoly.app.game.Stats
-import com.github.polypoly.app.game.User
+import com.github.polypoly.app.game.*
 import java.time.LocalDateTime
 import java.util.concurrent.CompletableFuture
 import java.util.concurrent.Future
+import kotlin.time.Duration.Companion.hours
 
 /**
- * A fake remote storage to test the functionalities without the database
+ * A fake remote storage to test the functionalities with compatibility for later use of the DB
  */
 class FakeRemoteStorage : IRemoteStorage {
```

Comment:
Why has FakeRemoteStorage become a storage for game lobbies, users, etc.? This looks like bad abstraction.